composer.json

```diff
@@ -36,15 +36,15 @@
         "sonata-project/notification-bundle": "^3.3",
         "sonata-project/seo-bundle": "^2.4",
         "sonata-project/user-bundle": "^3.6 || ^4.0",
-        "symfony/config": "^2.8",
-        "symfony/console": "^2.8",
-        "symfony/form": "^2.8",
-        "symfony/http-foundation": "^2.8",
-        "symfony/process": "^2.8",
-        "symfony/routing": "^2.8",
-        "symfony/security-bundle": "^2.8",
```

Comment:
Did you check if we can move security-bundle to security directly?

Security bundle has this 3 dependencies.

        "symfony/security": "~3.4|~4.0",
        "symfony/dependency-injection": "~3.4|~4.0",
        "symfony/http-kernel": "~3.4|~4.0"

I did a fast search, and we need dependencyInjection and httpKernel too. Can you add them please?